include/seqan3/alphabet/union_alphabet.hpp

```diff
@@ -129,13 +196,48 @@ struct union_alphabet
     //! the type of the alphabet when converted to char (e.g. via @link to_char @endlink)
     using char_type = typename first_alphabet_type::char_type;
     //! the type of the alphabet when represented as a number (e.g. via @link to_integral @endlink)
-    using integral_type = decltype(detail::at_least_type<value_size>());
+    using integral_type = detail::at_least_type_t<value_size>;
+    /*! the type used to assign a value from one of the base alphabets during
+     * copy construction or copy assignment. (i.e. via
+     * @link union_alphabet(const variant_type & alphabet) @endlink or
```

Comment:
here and in other places:
always use the syntax starting with `\` not with `@`
(I know it's not consistent throughout the files)